syz-ci/jobs.go

```diff
@@ -566,34 +566,40 @@ func (jp *JobProcessor) testPatch(job *Job, mgrcfg *mgrconfig.Config) error {
 	if err != nil {
 		return err
 	}
-	rep, err := aggregateTestResults(results)
+	ret, err := aggregateTestResults(results)
 	if err != nil {
 		return err
 	}
+	rep := ret.report
 	if rep != nil {
 		resp.CrashTitle = rep.Title
 		resp.CrashReport = rep.Report
-		resp.CrashLog = rep.Output
 	}
+	resp.CrashLog = ret.rawOutput
```

Comment:
Please update `dashboard/app/jobs_test.go` accordingly to fix how this looked in reply emails.